README.md

```diff
@@ -1,132 +1,184 @@
-- (CGFloat)randomFloatBetweenNumber:(CGFloat)minRange andNumber:(CGFloat)maxRange
-{
-    return ((float)arc4random() / ARC4RANDOM_MAX)
-    * (maxRange - minRange)
-    + minRange;
+// helper method
+- (NSString *)ordinalForInteger:(NSUInteger)integer {
+    NSString *ordinal = @"th";
+    if (integer % 10 == 1 && integer % 100 != 11) {
+        ordinal = @"st";
+    } else if (integer % 10 == 2 && integer % 100 != 12) {
+        ordinal = @"nd";
+    } else if (integer % 10 == 3 && integer % 100 != 13) {
+        ordinal = @"rd";
+    }
+    return ordinal;
 }
 ```
 
-Check out the documentation on NSMutableArray (Help -> Documentation and API Refrences).  NSMutableArray has some great methods ie. addObject, addObjectsFromArray, removeObject, removeObjectsFromArray 
+2 — Write the method implementation for `celebrateBirthday` to:
+
+  * increment the `age` property by one:
+    * `self.age++;`
+  * capture the return of calling `ordinalForInteger:` with the `self.age` property submitted as the argument:
+    * `NSString *ordinal = [self ordinalForInteger:self.age];`
+  * use the `stringWithFormat:` class method on `NSString` to create a formatted string that will read `HAPPY 30TH BIRTHDAY, MARK!!!` when an instance of `FISPerson` with the `name` property set to `@"Mark"` and the `age` property set to `29` performs the `celebrateBirthday` method; capture this formatted string into a new string object.
+  * use `NSLog()` to print the birthday message to the console, and
+  * finally, `return` the formatted string containing the birthday message so the test can verify it.
+
+### IV. Implement the `learnSkill...` Methods
+
+**Note:** *The strings in these method must be entered exactly in order for the tests to pass. Pay attention to capitalization and hyphenation.*
+
+1 — Write the method implementation for `learnSkillBash` to check whether the `self.skills` array already contains the string `@"bash"`, and if not, to add it.
+
+2 — Write the method implementation for `learnSkillXcode` to check whether the `self.skills` array already contains the string `@"Xcode"`, and if not, to add it. 
+
+3 — Write the method implementation for `learnSkillObjectiveC` to check whether the `self.skills` array already contains the string `@"Objective-C"`, and if not, to add it.
+
+4 — Write the method implementation for `learnSkillObjectOrientedProgramming` to check whether the `self.skills` array already contains the string `@"Object-Oriented Programming"`, and if not, to add it.
+
+5 — Write the method implementation for `learnSkillInterfaceBuilder` to check whether the `self.skills` array already contains the string `@"Interface Builder"`, and if not, to add it.
+
+### V. Implement the `qualifyAsTutor` Method
 
+Write the method implementation for the `qualifyAsTutor` method. It should check whether there are *at least* four elements in the `self.skills` array.
 
+  * if so, it should toggle the `qualifiedTutor` property to `YES` *and* return `YES`,
```

Comment:
Hm.. this is really weird... there's a method whose only responsibility is to toggle a property that is dependent on others? It feels like that logic should be baked into the `-learnSkill...` methods